src/SwiftTaskProvider.ts

```diff
@@ -132,12 +124,9 @@ export class SwiftTaskProvider implements vscode.TaskProvider {
     }
 
     /**
-     * Uses `swift package describe` to find all targets in this package.
+     * Uses SwiftPM package description cache to find all targets in this package.
      */
-    private async findTargets(): Promise<Target[]> {
-        const { stdout } = await exec('swift package describe --type json', { cwd: this.workspaceRoot });
-        return JSON.parse(stdout).targets.map((target: any) => {
-            return { name: target.name, type: target.type };
-        });
+    private findTargets(): Target[] {
```

Comment:
This can be removed now. We can just write `const targets = this.ctx.swiftPackage.targets;` where this method was used.

Reply:
@svanimpe I have a slightly different change which is in my LLDB branch, which adds a function to `SwiftPackage` that returns targets of a certain type (I needed it for the launch.json generation). Can this change wait until then?

Reply:
Sure!